snapshot/git/gitdb/bundlestore.go

```diff
@@ -199,9 +201,10 @@ func (s *bundlestoreSnapshot) Download(db *DB) error {
 		return nil
 	}
 
-	// TODO(dbentley): keep stats about bundlestore downloading
```

Comment:
while we're in this area of the code, can you add a comment (for go docs) explaining Download()....(document externally visible features):
Update's db's repo with this bundlestoreSnapshot's SHA. The snapshot must be a git 'bundle' file.  If necessary, and db.bundles.cfg.AllowStreamUpdate is True, the db's repo will also be updated with missing ?commits? needed to update db's repo with bundlestoreSnapshot's SHA.  Returns True if the repo contains the bundlestoreSnapshot's SHA, error otherwise.